src/server/pfs/cmds/cmds.go

```diff
@@ -328,7 +328,7 @@ Files can be read from finished commits with get-file.`,
 			if len(args) == 3 {
 				path = args[2]
 			}
-			fileInfos, err := pfsclient.ListFile(apiClient, args[0], args[1], path, "", shard())
+			fileInfos, err := pfsclient.ListFile(apiClient, args[0], args[1], path, "", shard(), true)
```

Comment:
So we're defaulting to a recursive list file? Does that behavior match what we had before?